contracts/governance/Staking/modules/StakingVestingModule.sol

```diff
@@ -49,16 +49,38 @@ contract StakingVestingModule is IFunctionsList, StakingShared {
      * TODO: remove - it was designed as a disposable function to initialize vesting checkpoints
      */
     function _setVestingStake(uint256 lockedTS, uint96 value) internal {
-        //delete all checkpoints (shouldn't be any during the first initialization)
+        require(
+            lockedTS > kickoffTS,
+            "Invalid lock dates: must greater than contract creation timestamp"
+        );
+
+        // locked date must be multiples of 14 days
+        require(
+            (lockedTS - kickoffTS) % 14 days == 0,
```

Comment:
please use the TWO_WEEKS constant which used everywhere else in the contract

Reply:
Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/472/commits/871541179e096327e612ede25477204ffae4dc87)